plugins/woocommerce/includes/abstracts/abstract-wc-payment-gateway.php

```diff
@@ -562,3 +562,79 @@ public function add_payment_method() {
 		);
 	}
 }
+
+/**
+ * Email the site admin when a payment gateway is enabled.
+ *
+ * @param string $option Option name.
+ * @param array  $old_value Old value.
+ * @param array  $value New value.
+ * @since 8.4.0
+ */
+function wc_notify_payment_gateway_enabled( $option, $old_value, $value ) {
```

Comment:
Following up on an earlier conversation, I also think this may not be the ideal location for a top-level function. We probably have a few cases like this (especially within the `includes` dir) where loading classfiles come with side-effects, but I think it's something we want to veer away from.

Reply:
Addressed. (Will push in a bit.)